espnet2/enh/loss/criterions/time_domain.py

```diff
@@ -207,3 +209,65 @@ def forward(
         )
 
         return si_snr
+
+
+class TimeDomainMSE(TimeDomainLoss):
+    def __init__(self):
+        super().__init__()
+
+    @property
+    def name(self) -> str:
+        return "time_MSE_loss"
+
+    def forward(self, ref, inf) -> torch.Tensor:
+        """Time-domain MSE loss forward.
+
+        Args:
+            ref: (Batch, T) or (Batch, T, C)
+            inf: (Batch, T) or (Batch, T, C)
+        Returns:
+            loss: (Batch,)
+        """
+        assert ref.shape == inf.shape, (ref.shape, inf.shape)
+
+        mseloss = (ref - inf).pow(2)
+        if ref.dim() == 3:
+            mseloss = mseloss.mean(dim=[1, 2])
+        elif ref.dim() == 2:
+            mseloss = mseloss.mean(dim=1)
+        else:
+            raise ValueError(
+                "Invalid input shape: ref={}, inf={}".format(ref.shape, inf.shape)
+            )
+        return mseloss
+
+
+class TimeDomainL1(TimeDomainLoss):
+    def __init__(self):
+        super().__init__()
+
+    @property
+    def name(self) -> str:
+        return "time_L1_loss"
```

Comment:
Not sure. Is it better to use time_domain_L1_loss or TD_L1_loss?

Reply:
I think so. I will modify it. Thanks!